packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -1031,58 +980,197 @@ class RawGestureDetectorState extends State<RawGestureDetector> {
       final List<String> gestures = _recognizers.values.map<String>((GestureRecognizer recognizer) => recognizer.debugDescription).toList();
       properties.add(IterableProperty<String>('gestures', gestures, ifEmpty: '<none>'));
       properties.add(IterableProperty<GestureRecognizer>('recognizers', _recognizers.values, level: DiagnosticLevel.fine));
+      properties.add(DiagnosticsProperty<bool>('excludeFromSemantics', widget.excludeFromSemantics, defaultValue: false));
+      properties.add(DiagnosticsProperty<SemanticsGestureDelegate>('semantics', widget.semantics));
```

Comment:
what does this look like when you print this object?

Reply:
(maybe add a test for this... would be good to test this stuff more anyway)

Reply:
Added tests in https://github.com/flutter/flutter/pull/33936/commits/7c87ae95d40101ecfe17630479a394ba926762b5